lib/questionlib.php

```diff
@@ -2594,3 +2076,726 @@ function get_next_version(int $questionbankentryid): ?int {
 
     return null;
 }
+/**
+ * Get all questions that belong to a category.
+ *
+ * @param $categoryid int Category id.
+ * @return null|array Questions in a category.
+ * @throws dml_exception
+ */
+function get_questions_in_category(int $categoryid): ?array {
```

Comment:
this method is not used anywhere rather than the questionlib_test, if that's the case, this function need to be created inside the test file, not in the questionlib as this totally works like a helper for unit test